students/N0vA/lesson_05/mailroom3.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+#### Mailroom Project Part 2 ####
+
+import sys
+import pathlib
+
+# Create list of donors and their donation history
+database = {'Bill Gates': [2000000, 250000000], 
+            'Jeff Bezos': [2000000],
+            'Elon Musk': [50000000, 10000000], 
+            'Howard Schultz': [1000000], 
+            'Paul Allen': [450000000]}
+
+
+# Send a thank you
+def thank_you():
+    
+    # Set up inputs for appending database
+    name = 'list'
+    while name == 'list':
+        name = input("Alright.  Which donor would you like to send a thank you card?\nType 'list' to see a list of past donors >>>")   
+        if name == 'list':
+            print(list(database.keys()))
+    
+    # Enter donation amount
+    amount = input('How much was their donations? ')
+    try: 
+        amount = int(amount)
+    except ValueError:
+        amount = input('Enter a valid donation amount: ')
+        amount = int(amount)
```

Comment:
what if they enter invalid number for the second time? 

your program will crash, ideally you want this prompt to be inside a while True loop that exists once a valid number is entered (int() cast passes)